gh-pr-context.py

```
@@ -656,13 +682,140 @@ def cmd_monitor(argv):
     if sub_command == "status":
         cmd_monitor_status(rest)
+    elif sub_command == "comments":
+        cmd_monitor_comments(rest)
     elif sub_command in ("-h", "--help"):
         usage_monitor()
         sys.exit(0)
     else:
         die(f"unknown monitor sub-command: {sub_command}")
 
 
+def cmd_monitor_comments(argv):
```

Ruff (0.15.15)

[warning] Too many branches (30 > 12)

(PLR0912)